data/init.sql

```diff
@@ -152,4 +152,14 @@ CREATE TABLE IF NOT EXISTS course_sections(
 	course_section_name VARCHAR(60),
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP
-)
+);
+
+CREATE TABLE IF NOT EXISTS course_offerings_instructors(
+	offering_id INT(11) UNSIGNED NOT NULL,
+	user_id INT(11) UNSIGNED NOT NULL,
+	FOREIGN KEY (user_id) REFERENCES users(id),
+	FOREIGN KEY(offering_id) REFERENCES course_offerings(id) ON DELETE CASCADE,
+	created_at TIMESTAMP,
+	updated_at TIMESTAMP,
+	PRIMARY KEY (offering_id, user_id)
+);
```

Comment:
We could make the PK unique

Reply:
It should be unique though, because its a combination of 2 unique keys no? I mena, we could add a seperate ID key, but I feel we would never use it and it would be redundant no?

Reply:
actually forget what I said, I got confused with something else